projects/robots/festo/robotino3/protos/Robotino3Webcam.proto

```diff
@@ -0,0 +1,239 @@
+#VRML_SIM R2020a utf8
+# license: Apache License 2.0
+# license url: http://www.apache.org/licenses/LICENSE-2.0.
+# This is a PROTO file for Webots of a webcam for the robot Robotino 3 from Festo.
+# It is a feature for Robotino 3 to fullfil your requirement.
+# The model is based on the Logitech® HD Pro Webcam C920.
+
+PROTO Robotino3Webcam [
+  field  SFVec3f     translation    0 0 0
+  field  SFRotation  rotation       0 1 0 0
+  field  SFString    name           "Webcam for Robotino 3"
+  field  SFString    model          "Festo - Robotino's webcam"
```

Comment:
We should ask some fields, at least:
  - cameraName
  - cameraWidth  (which should be bigger than the default of 64, e.g. 640)
  - cameraHeight  (which should be bigger than the default of 64, e.g. 480)